robot_controllers/src/cartesian_twist.cpp

```diff
@@ -164,12 +166,26 @@ void CartesianTwistController::update(const ros::Time& now, const ros::Duration&
   if (!initialized_)
     return;  // Should never really hit this
 
+  if(fksolver_->JntToCart(tgt_jnt_pos_,cartPose) < 0)
+  {
+    cartPose = KDL::Frame::Identity();
+    ROS_ERROR_THROTTLE(1.0, "FKsolver solver failed");
+  }
+
   // Copy desired twist and update time to local var to reduce lock contention
   KDL::Twist twist;
   ros::Time last_command_time;
-  {
+  { // what is this?? no functin? what is this bracket for?
```

Comment:
this bracket is for scoping the mutex on the next line -- we should remove this comment